readthedocs/doc_builder/python_environments.py

```diff
@@ -498,7 +500,7 @@ def _install_mamba(self):
 
     def setup_base(self):
         conda_env_path = os.path.join(self.project.doc_path, 'conda')
-        version_path = os.path.join(conda_env_path, self.version.slug)
+        os.path.join(conda_env_path, self.version.slug)
```

Comment:
This line seems can be completely removed because we are not assigning it to any variable.